gdbgui/src/js/Actions.js

```diff
@@ -143,8 +190,49 @@ const Actions = {
     Actions.inferior_program_exited();
     GdbApi.run_gdb_command([`-target-select remote ${user_input}`]);
   },
-  remote_connected() {
+  connect_to_gdbserver_mpi(user_input) {
+    // https://sourceware.org/gdb/onlinedocs/gdb/GDB_002fMI-Target-Manipulation.html#GDB_002fMI-Target-Manipulation
+    store.set("source_file_paths", []);
+    store.set("language", "c_family");
+    store.set("inferior_binary_path", null);
+    store.set("process_on_focus", 0);
+    store.set("is_mpi", true);
+    // create a bigger padding to draw processor on focus selector
+    Actions.inferior_program_exited();
+    // parse user input
+    let host_port_arr = user_input.split(":");
+
+    let data_lines;
+    // Check the server names from
+    if (host_port_arr[0] === "*") {
+      jQuery.ajax({
+        url: "mpi_processes_info",
+        success: function(data, status) {
+          data_lines = data.split(/\r?\n/);
+        },
+        async: false
+      });
+    }
```

Comment:
I am going to eventually replace all jQuery.ajax calls with [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch). 

It would be helpful if this was switched to use fetch, and the function turned into an `async` function. If you don't know what that means, don't worry about it. I will just fix it in a future update.

Reply:
I will  have a look. My programming skill in python and Javascript ( now TypeScript ) are basic, I am learning while going.

Reply:
It’s fine to leave as is. I can migrate it along with the other calls at a later time.